docs/data/joy/pages.ts

```diff
@@ -67,7 +67,11 @@ const pages = [
       {
         pathname: '/joy-ui/components/surfaces',
         subheader: 'surfaces',
-        children: [{ pathname: '/joy-ui/react-card' }, { pathname: '/joy-ui/react-sheet' }],
+        children: [
+          { pathname: '/joy-ui/react-card' },
+          { pathname: '/joy-ui/react-sheet' },
+          { pathname: '/joy-ui/react-grid' },
```

Comment:
I think this should be under `layout` if you take a look at Material UI.

<img width="277" alt="image" src="https://user-images.githubusercontent.com/18292247/206347852-748b3ff9-4efe-4a61-8b23-b659e77bd525.png">